public/index.html

```diff
@@ -19,6 +19,25 @@
     />
     <title>HNScan | Block Explorer for Handshake</title>
 
+    <!-- Solitude - Be Alone on The Web Again -->
+    <script>
+      !function (a, l, o, n, e) {
+        (a[o] = a[o] || function () {
+          (a[o].q = a[o].q || []).push(arguments)
+        });
+        a[o].l = +new Date;
+        n = l.createElement('script');
+        e = l.scripts[0];
+        n.src = '//cdn.urkellabs.com/solitude.js';
+        e.parentNode.insertBefore(n, e)
+      }(window, document, 'solitude');
+      // @todo change to production api url
+      // @todo move this line to the tracking script
+      solitude('setTrackerUrl', 'http://localhost:9090/collect')
```

Comment:
We should either use production here or not include since it'll throw errors once merged and deployed.